template/.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,72 @@
+      - name: Cache Pre-commit hooks
+        uses: actions/cache@v4.2.4
+        env:
+          cache-name: cache-pre-commit-hooks
+        with:
+          path: ${{ env.PRE_COMMIT_HOME }}
+          key: ubuntu-24.04-py${{ inputs.python-version }}-node-${{ inputs.node-version}}-${{ env.cache-name }}-${{ hashFiles('.pre-commit-config.yaml') }}
+          restore-keys: |
+            ubuntu-24.04-py${{ inputs.python-version }}-node-${{ inputs.node-version}}-${{ env.cache-name }}-
+
+      - name: Run pre-commit
+        run:  pre-commit run -a
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)